sdk/api/metalava.txt

```diff
@@ -582,15 +582,28 @@ package com.mapbox.maps.viewannotation {
     method public android.view.View? getViewAnnotationByFeatureId(String featureId);
     method public com.mapbox.maps.ViewAnnotationOptions? getViewAnnotationOptionsByFeatureId(String featureId);
     method public com.mapbox.maps.ViewAnnotationOptions? getViewAnnotationOptionsByView(android.view.View view);
+    method public com.mapbox.maps.viewannotation.ViewAnnotationUpdateMode getViewAnnotationUpdateMode();
     method public void removeAllViewAnnotations();
     method public void removeOnViewAnnotationUpdatedListener(com.mapbox.maps.viewannotation.OnViewAnnotationUpdatedListener listener);
     method public boolean removeViewAnnotation(android.view.View view);
+    method public void setViewAnnotationUpdateMode(com.mapbox.maps.viewannotation.ViewAnnotationUpdateMode mode);
     method public boolean updateViewAnnotation(android.view.View view, com.mapbox.maps.ViewAnnotationOptions options);
+    field public static final com.mapbox.maps.viewannotation.ViewAnnotationManager.Companion Companion;
+  }
+
+  public static final class ViewAnnotationManager.Companion {
```

Comment:
we can get rid of companion by adding `JvmField` annotation to the `DEFAULT_UPDATE_MODE`